test/hotspot/jtreg/compiler/c2/irTests/TestIfMinMax.java

```diff
@@ -295,7 +295,7 @@ public void checkTestMinIntReduction(Object[] vals) {
     }
 
     @Test
-    @IR(applyIf = { "SuperWordReductions", "true" },
+    @IR(applyIfAnd = { "SuperWordReductions", "true", "MaxVectorSize", ">=32" },
```

Comment:
Maybe we should only add this new requirement for RVV? Seems that it is not needed for avx512.

Reply:
Thanks for the review! I will add restrictions only for RISC-V.